platform/engineering/frontend/speedcurve/user-guide.md

```diff
@@ -0,0 +1,151 @@
+# Monitoring performance on VA.gov
+
+SpeedCurve provides insight into the interaction between performance and design to help Veteran-facing services (VFS) teams deliver fast and engaging user experiences.
+
+SpeedCurve also provides performance feedback of each app such as page size and how long it takes to load a page, and can show you the impact new features have on the performance of the app.
+
+SpeedCurve's dashboards provide various perspectives and options. VFS teams can define and monitor performance metrics across VA.gov, test performance in the context of continuous deployment, test responsiveness, and analyze assets. VFS teams can also set a performance budget so teams can see and track when the performance of the site lags.
+
+These are some of SpeedCurve features:
+
+- Track dozens of metrics, including custom metrics.
+- Create performance budgets and get alerts.
+- Diagnose performance issues at the page level (Provided with Google's Lighthouse & PageSpeed Insights audits, there is a list of performance rules, stacked and color-coded in their order of importance.)
+- Analyze the performance impact of ads and third parties.
+
+## Getting started
+
+One user from each veteran-facing services (VFS) team is designated as the administrator for the team. If you are designated as the administrator, you will receive an invitation via email to sign up for SpeedCurve.
+
+Once you are signed in to SpeedCurve, perform the following tasks to configure your team's settings:
+
+1. [Create a team](#create-a-team)
+2. [Add team members](#add-team-member)
```

Comment:
Did you know that Markdown will render the correct list item number in an ordered list for you even if you type the wrong number?

For example:
```
// Markdown
1. [Create a team](#create-a-team)
7. [Add team members](#add-team-member)
4. [Configure default settings](#default-settings)

// Output
1. [Create a team](#create-a-team)
2. [Add team members](#add-team-member)
3. [Configure default settings](#default-settings)
```

Source: https://www.markdownguide.org/basic-syntax/#ordered-lists

I'm not recommending you make that change here. I'm more just letting you know so you can avoid a tedious list update if you have to add more steps to any of these lists in the future.

Reply:
No, I didn't. Thanks for letting me know